src/components/ContentNode/Reference.vue

```diff
@@ -43,7 +43,7 @@ export default {
       return name !== notFoundRouteName;
     },
     isSymbolReference() {
-      return this.kind === 'symbol'
+      return this.kind === 'symbol' && !this.hasInlineFormatting
```

Comment:
What happens if a `overridingTitleInlineContent` has a kind of `codeVoice`? Won't it be using the codeVoice component then?

Reply:
@marinaaisa If `overridingTitleInlineContent` is present, it will pass over any special formatting of the content, to the content itself, by rendering it via `renderChildren`.

Here Marcus just ensures the Reference uses the `ReferenceInternal` instead of the `ReferenceInternalSymbol` which just adds an extra `CodeVoice` component.

Reply:
In that scenario, the codeVoice would still be applied through the slot, but it wouldn't get the extra code voice that we were previously applying here.

You can test this example in the fixture I provided.

In your PR, this scenario resulted in a nested code element: `<code><code>...` (one from the content itself and another from the result of this logic applying another.

Reply:
Okay, I was missing that. Makes sense, thanks!